plugins/digitalbitbox/digitalbitbox.py

```diff
@@ -4,7 +4,7 @@
 #
 
 try:
-    from electroncash.bitcoin import TYPE_ADDRESS, push_script, var_int, msg_magic, Hash, verify_message, pubkey_from_signature, point_to_ser, public_key_to_p2pkh, EncodeAES, DecodeAES, MyVerifyingKey, int_to_hex
+    from electroncash.bitcoin import TYPE_ADDRESS, push_script, var_int, msg_magic, Hash, verify_message, pubkey_from_signature, point_to_ser, public_key_to_p2pkh, EncodeAES, DecodeAES, MyVerifyingKey, int_to_hex, hmac_oneshot, EncodeAES_bytes, DecodeAES_bytes
```

Comment:
Would you mind terribly removing the import for `EncodeAES` and `DecodeAES`?  I'm trying to deprecate those functions to be in-line with the upstream APIs. :)

If you're too busy let me know and I'll just merge as-is and change it myself later..